### PR TITLE
feat:added hamburger

### PR DESCRIPTION
**Pull request by tushargoyal812**
https://user-images.githubusercontent.com/69681896/155145664-360b950a-2701-4260-a40a-b9b7c3d993a1.mp4

### CODE REVIEW

File: components/hamburger/hamburger.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+// hamburger.addEventListener("click",()=>{
+//     crossBtn.style.display='block'
+//     hamburger.style.display='none';
+//     if(showUI.style.left ==="0px"){
+//         showUI.style.left="-100%"
+//     }else{
+//         showUI.style.left="0px"
+//     }
+// })
```
Comment (Aditi0505) [inline components/hamburger/hamburger.js:25]: Please remove commented code if not needed.

**Review by Aditi0505 — APPROVED** [review]
Left one suggestion. Rest looks good to me.

**Review by kotesh-arya — APPROVED** [review]
Removing the commented code-blocks is a good practice to be followed, the rest of the code
Looks good for a merge 👍🏻

File: components/hamburger/hamburger.js (hunk 1 above)
Comment (kotesh-arya) [inline components/hamburger/hamburger.js:25]: Removing the commented code is a good practice to be followed

**Pull request merged**